lib/manageiq/cross_repo/runner.rb

```diff
@@ -3,9 +3,9 @@
 
 module ManageIQ::CrossRepo
   class Runner
-    attr_reader :test_repo, :core_repo, :gem_repos, :script_cmd
+    attr_reader :test_repo, :core_repo, :gem_repos, :script_cmd, :test_suite
```

Comment:
Minor, but for parity

```suggestion
    attr_reader :test_repo, :core_repo, :gem_repos, :test_suite, :script_cmd
```